vphysics_jolt/vjolt_collide.cpp

```diff
@@ -619,7 +619,9 @@ void JoltPhysicsCollision::VCollideLoad( vcollide_t *pOutput, int solidCount, co
 		if ( pCollideHeader->vphysicsID != ivp_compat::VPHYSICS_COLLISION_ID ||
 			 pCollideHeader->version != ivp_compat::VPHYSICS_COLLISION_VERSION )
 		{
-			Log_Warning( LOG_VJolt, "Skipped solid %d due to invalid header (id: %.4s, version: 0x%x)\n", i, &pCollideHeader->vphysicsID, pCollideHeader->version );
+			char szId[5] {};
+			ID_TO_STRING( pCollideHeader->vphysicsID, szId );
+			Log_Warning( LOG_VJolt, "Skipped solid %d due to invalid header (id: %.4s, version: 0x%x)\n", i, szId, pCollideHeader->version );
```

Comment:
What is the problem here exactly? This is functionally identical.

Reply:
I have changed this code anyway so it won't print the ID in another PR https://github.com/Joshua-Ashton/VPhysics-Jolt/pull/38 so probably fine to just drop this part.

Reply:
The previous code is fine but trips a warning because the format specifies string, but it's being fed `int*`. I'll drop this though, as suggested